Automattic/jetpack
# Connection UI: initial state

#### Changes proposed in this Pull Request:
- `Initial_State` class for the package
- adding the `connectionStatus` data into the initial state

#### Jetpack product discussion
p1HpG7-bmK-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Go to "Tools -> Connection Manager". You should see the "Connection Manager" title, there's no other content.
2. Open the browser console and see the value of the global variable `CUI_INITIAL_STATE`. It should reflect the connection status:
<img width="556" alt="Screen Shot 2021-03-30 at 12 06 55 PM" src="https://user-images.githubusercontent.com/1341249/113191999-daf60900-9223-11eb-986d-a3b9792f3b17.png">
